client/components/chart/legend.jsx

```diff
@@ -36,25 +23,21 @@ const LegendItem = React.createClass( {
 			</li>
 		);
 	}
+}
 
-} );
-
-const Legend = React.createClass( {
-	displayName: 'ModuleChartLegend',
+LegendItem.propTypes = {
```

Comment:
Technically it's ES7, but you can define propTypes inside the component, see the [Property initializers section here](https://babeljs.io/blog/2015/06/07/react-on-es6-plus).